src/elements/Chip/DeletableChip.tsx

```diff
@@ -18,17 +18,23 @@ export interface DeletableChipProps extends Omit<ChipProps, 'end'> {
 }
 
 export const DeletableChip: FC<DeletableChipProps & ChipRef> = forwardRef(
-  ({ children, disabled, deleteIcon, onDelete, size, ...rest }, ref) => (
+  ({ children, disabled, deleteIcon, onDelete, size, color, ...rest }, ref) => (
     <Chip
       ref={ref}
       size={size}
       disabled={disabled}
+      color={color}
       end={
         <Button
           tabIndex={0}
           variant="text"
           size={size}
-          className={classNames(css.deleteButton, { [css[size]]: true })}
+          className={classNames(
+            css.deleteButton,
+            { [css[size]]: true },
+            { [`text-${color}-500`]: rest?.variant === 'outline' },
+            { ['text-white']: rest?.variant === 'filled' }
+          )}
```

Comment:
this is maybe a little hacky, but it ensures the close button matches the color when using the outline variant
![image](https://github.com/reaviz/reablocks/assets/40581813/339a4af4-609e-4d52-8dc2-a7df7c05b996)

Reply:
nvm replaced this with `text-inherit`